pkg/packet/utilities.go

```diff
@@ -0,0 +1,112 @@
+package packet
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+// {
+// 	"ips": [
+// 	  "10.144.144.144",
+// 	  "10.144.145.66"
+// 	],
+// 	"name": "volume-4b6ed3d8",
+// 	"capacity": {
+// 	  "size": "100",
+// 	  "unit": "gb"
+// 	},
+// 	"iqn": "iqn.2013-05.com.daterainc:tc:01:sn:b06f15a423fec58b"
+// }
+
+// PacketCapacityMetaData exists for parsing json metadata
+type PacketCapacityMetaData struct {
+	Size string `json:"size"`
+	Unit string `json:"unit"`
+}
+
+// PacketVolumeMetadata exists for parsing json metadata
+type PacketVolumeMetadata struct {
+	Name     string                 `json:"name"`
+	Ips      []string               `json:"ips"`
+	Capacity PacketCapacityMetaData `json:"capacity"`
+	Iqn      string                 `json:"iqn"`
+}
+
+// get all the metadata, extract only the parsed volume information, select the desired volume
+func GetPacketVolumeMetadata(volumeName string) (PacketVolumeMetadata, error) {
+
+	empty := PacketVolumeMetadata{}
+
+	res, err := http.Get("https://metadata.packet.net/metadata")
```

Comment:
why not use packngo/metadata for this?

Reply:
I didn't realize it existed :( Obviously packngo/metadata is preferred.  Will replace.

Reply:
:+1:  just wondering if there was a deficiency in packngo/metadata

Reply:
I have a fix and a test case, will try to get that pushed through.